### PR TITLE
Try windows build

### LINKED ISSUE #9: Build Windows from source?
Seems we are just copying binaries over for our Windows package. Would be really awesome if we built this from source. Hopefully, that would make it easier to extend the functionality in this package as well by adding needed dependencies.

### PR DESCRIPTION
**Pull request by jschueller**
Fixes #9

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,55 +6,61 @@ package:
...
+    - flang-fix.patch
+    - cmake-3.10-boost-1.66-support.patch
 
 build:
-  number: 2
+  skip: True  # [win and not py36]
+  number: 3
   detect_binary_files_with_prefix: true
+  features:
+    - vc14  # [win and py>=35]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Might want to try `msvc_compiler` as we may need to build against a particular compiler, but we don't want to constrain where `cmake` can be installed as nothing is going to link to it. Things will only use `cmake` to build.

Comment (jschueller) [reply recipe/meta.yaml]: Thanks, trying now.

Comment (isuruf) [reply recipe/meta.yaml]: That is if you bundle your dependencies. But in this PR, dependencies are used from other packages. So, you need this

Comment (jschueller) [reply recipe/meta.yaml]: Oh, maybe that's not what we want, maybe I should not use dependencies then.

**Pull request merged**